src/install-deps.js

```diff
@@ -0,0 +1,37 @@
+const path = require("path");
+const glob = require("glob");
+const cp = require("child_process");
+const parallelLimit = require("run-parallel-limit");
```

Comment:
If you prefer to use a promise version of this too, thats fine with me.  Just throwing that out there.

Reply:
Not familiar with `p-` libs. If you want to update go for it.

I promisified the main API